lean/commands/create_project.py

```diff
@@ -293,8 +294,7 @@ def create_project(name: str, language: str) -> None:
     full_path = Path.cwd() / name
 
     if not container.path_manager.is_path_valid(full_path):
-        raise MoreInfoError(f"'{name}' is not a valid path",
-                            "https://www.lean.io/docs/v2/lean-cli/key-concepts/troubleshooting#02-Common-Errors")
```

Comment:
Maybe we can keep the link to `"https://www.lean.io/docs/v2/lean-cli/key-concepts/troubleshooting#02-Common-Errors"` 👍